service/src/main/java/com/epam/esm/data/GiftCertificateDto.java

```
@@ -0,0 +1,56 @@
+package com.epam.esm.data;
+
+import com.epam.esm.entities.Tag;
+import com.epam.esm.validator.DtoTag;
+import com.epam.esm.validator.DtoTagValidator;
+import com.epam.esm.validator.PatchDto;
+import com.epam.esm.validator.SaveDto;
+import com.fasterxml.jackson.annotation.JsonFormat;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.validation.annotation.Validated;
+
+import javax.validation.Valid;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Null;
+import javax.validation.constraints.Size;
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+import java.util.List;
+
+import static com.epam.esm.exception.ExceptionCodes.*;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class GiftCertificateDto {
```

What happens if I try to create a new entity with an empty name and incorrect price and duration fields?
Will I get all three errors at once or just one?

---

The first error will be shown. I turned on fail fast mode to increase perfomance in case of large objects.